src/app/components/mno-offers-list/mno-offers-list.html

```diff
@@ -15,18 +15,18 @@
     <tr ng-repeat="offer in offers">
       <td>{{offer.name}}</td>
       <td>{{pricingType(offer)}}</td>
-      <td ng-if="offer.prices">
+      <td ng-if="offer.prices && !offer.quote_based">
```

Comment:
We should probably extract this to a method in the controller as it's used 4 times!
It also make the negation easier to use (reduce error due to bad boolean algebra ;) )